.travis.yml

```diff
@@ -46,18 +46,6 @@ jobs:
 
   allow_failures:
 
-    - stage: test
-      name: 'Rubocop'
-      script:
-        - bundle exec rubocop
```

Comment:
Don't we want this in travis?

Reply:
The rubocop check? Yes. We are not executing any test in GitLab. It's all done in Travis. I removed the rubocop test in the allow_failures section but it's still in the normal section. Meaning that we shouldn't be able to merge if the test is failing.

See: https://github.com/DigitalNZ/supplejack_manager/blob/2dc949617e298727bcf1d197eb733fcb2a28e8a1/.travis.yml